### ISSUE TITLE
Debugger freezes in VSCode

### ISSUE THREAD

**Issue by lothas** [microsoft/debugpy#108]
I'm trying to debug a piece of code and experiencing a weird bug:
I run my code up to a breakpoint and then start stepping through it (with F10/F11). I sometimes hover over some variables to check the content. At any (seemingly random) point, VSCode stops stepping over the code. No error is thrown. The only thing left for me to do is stop debugging (hit the stop button) and start the run again.
[debugger.vscode_f0ea2b1b-f0b2-43e2-9596-a42593fe7528.log](https://github.com/microsoft/vscode-python/files/4380102/debugger.vscode_f0ea2b1b-f0b2-43e2-9596-a42593fe7528.log)

**Comment by int19h** [comment]
Okay, so this wasn't actually fixed by #83, because the log grabs sys.stderr before it is redirected to /dev/null.

**Comment by Amuri44** [comment]
> @TimotheusT @ftrofin @lothas Thank you for the logs. We are trying to narrow down this issue. Here is something you can do temporarily to unblock. Add this to your user settings. ![image](https://user-images.githubusercontent.com/3840081/78059410-a7498000-733e-11ea-9b27-48e75913fc3c.png)
> 
> ```js
> "python.experiments.optOutFrom": [
>     "PtvsdWheels37 - experiment"
> ],
> ```
> 
> Re-load after adding this setting. This should get you unblocked.

How is about the the code to add for debugging julia code?

**Comment by icc-jhanvillasin** [comment]
Im currently having the same issue but in AL Extension Business Central. anyone can help me ?

**Comment by SohailChohan143** [comment]
> Im currently having the same issue but in AL Extension Business Central. anyone can help me ?

Same here please if you get solution pls pin me too

**Comment by icc-jhanvillasin** [comment]
anyone can help?

**Comment by SohailChohan143** [comment]
Use older veersion of VS code fix my issue "VSCodeUserSetup-x64-1.74.3"

**Comment by Liqs-v2** [comment]
This is still happening to me

**Comment by rchiodo** [comment]
If you're experiencing an issue with the debugger freezing, please open a new issue.

**Comment by pumbatimon473** [comment]
Looks like I am not the only one, and this problem has more than 5 years of history.

Debugging Java code is really slow and freezes while analyzing the values of the local objects.

Did not expect that from a development tool.

**Comment by tombohub** [comment]
Debugger with django stops responding.

I dont see proposed solution https://github.com/microsoft/debugpy/issues/108#issuecomment-606784403:

<img width="483" height="223" alt="Image" src="https://github.com/user-attachments/assets/9540cd52-938f-4625-9701-a5b317d1e940" />

**Comment by rchiodo** [comment]
@tombohub I recommend you open a new issue. Any comments made above are very out of date.
